package.json

```diff
@@ -5,8 +5,7 @@
   "main": "index.js",
   "scripts": {
     "prepublishOnly": "yarn build",
-    "build": "tsc",
-    "postinstall": "yarn build"
```

Comment:
The index.JS file should not be committed but rather the following script should be added

`"prepare": "npm build"`

`prepare` Automatically runs when installing a package from a Github link.

Reply:
Got it, trying that out now. 
When I tried pre-install, the project was throwing typescript errors.
Will see if prepare gives different results.

Reply:
When attempting prepare this error comes back 
![image](https://user-images.githubusercontent.com/5866571/106550097-5149e980-64c7-11eb-9ea2-0d2bbb05579f.png)

This is happening because "react-native-reanimated" is a peer-dependency in the project. 

Given that, I do not see any other modules that depend on react-native-reanimated. 
Going to move it to devDependencies and see if the installation is smooth from there.

Reply:
That worked and I just learned something. Thanks @paullinator !